frontend/src/lib/aspidaClient.ts

```diff
@@ -4,7 +4,7 @@ import api from "../../api/$api";
 const fetchConfig = (token: string) => {
   return {
     // credentials: "include",
-    baseURL: "http://localhost:9000",
+    baseURL: process.env.NEXT_PUBLIC_API_BASE_URL,
```

Comment:
```suggestion
    baseURL: process.env.API_BASE_URL,
```
.env.templateで言った内容と同じ！